### PR TITLE
Add example solidjs-with-layout

### PR DESCRIPTION
**Pull request by noveogroup-amorgunov**
Hello guys. I think this repo should contain smaller examples which preset how to work with FSD. Review please example show how to work with layouts. If this format is OK, I want to add a lot of examples how work with api, store, router, i81n, modal-manager, user-session and so on.

- Update README.md
- Add monorepo with `pnpm`
- Add example `solidjs-with-layout` - see live demo https://stackblitz.com/github/noveogroup-amorgunov/feature-sliced--examples/tree/feature/add-example-solidjs-with-layout/examples/solidjs-with-layout?file=README.md

Example based on SolidJS and you may have a question - why? I chose this one for two reasons: take a little part in popularizing the library and use different from popular ui-frameworks (react/vue/svelte/angular) to show that FSD can be used as framework-agnostic solution.

### CODE REVIEW

**Comment by noveogroup-amorgunov** [comment]
@illright @unordinarity hello, see please this pull request

File: examples/solidjs-with-layout/src/app/layouts/BaseLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import type { RouteSectionProps } from '@solidjs/router'
+import { Layout } from '@/shared/ui'
+import { Header } from '@/widgets/header'
+
+type Props = RouteSectionProps
+
+/**
+ * ✅ FSD Best practice:
+ *
+ * (1) Devide layout in two modules: dumb layout grid (`@shared/ui/Layout/*`)
```
Comment (illright) [inline examples/solidjs-with-layout/src/app/layouts/BaseLayout.tsx:10]:
```suggestion
 * (1) Divide layout in two modules: dumb layout grid (`@shared/ui/Layout/*`)
```

File: examples/solidjs-with-layout/src/app/layouts/BaseLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * and smart layout with widgets (this file)
+ *
+ * (2) Avoid cross-import with dependency inversion via render prop pattern
```
Comment (illright) [inline examples/solidjs-with-layout/src/app/layouts/BaseLayout.tsx:13]:
```suggestion
 * (2) Avoid importing from higher layers with dependency inversion via the render prop pattern
```

File: examples/solidjs-with-layout/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## solidjs-with-layout
+
+This example shows how to work work with layout (or layouts). Split layout to dumb component (with markup) and smart component for widget compositions. Dumb layout can be placed in `@share/ui/Layout`, smart - `@app/layouts/BaseLayout.tsx`. See [@app/layouts/baseLayout.tsx]() and [@shared/ui/Layout/*]() in code.
```
Comment (illright) [inline examples/solidjs-with-layout/README.md:3]:
```suggestion
This example shows how to work work with layout (or layouts). Split layout to dumb component (with markup) and smart component for widget compositions. Dumb layout can be placed in `@shared/ui/Layout`, smart - `@app/layouts/BaseLayout.tsx`. See `@app/layouts/baseLayout.tsx` and `@shared/ui/Layout/*` in code.
```

File: README.md [hunk 4]
```diff
@@ -1,17 +1,12 @@
...
 
-## Inner
+## List
 
-- [TodoApp](./todo-app)
+- **[examples/react-effector-todo-app](./todo-app)** <sub>[outdated]</sub>
+- **[examples/solidjs-with-layout](./examples/solidjs-with-layout)** - shows how works with layout
 
-## RealWorld
+## Community fully examples
```
Comment (illright) [inline README.md:10]:
```suggestion
## Community examples — full projects
```

**Review by illright — APPROVED** [review]
Hi, thanks a lot! I only have a few minor comments regarding grammar here and there, but the project itself is good. I also like how you set up this repository, great job!

File: examples/solidjs-with-layout/src/app/layouts/BaseLayout.tsx (hunk 1 above)
Comment (noveogroup-amorgunov) [reply examples/solidjs-with-layout/src/app/layouts/BaseLayout.tsx]: Fixed 27d7518165bc794051ef17738f2f31da00e5748d

File: examples/solidjs-with-layout/src/app/layouts/BaseLayout.tsx (hunk 2 above)
Comment (noveogroup-amorgunov) [reply examples/solidjs-with-layout/src/app/layouts/BaseLayout.tsx]: Fixed 27d7518165bc794051ef17738f2f31da00e5748d

File: examples/solidjs-with-layout/README.md (hunk 3 above)
Comment (noveogroup-amorgunov) [reply examples/solidjs-with-layout/README.md]: Fixed (add backticks and correct link to files) 27d7518165bc794051ef17738f2f31da00e5748d

File: README.md (hunk 4 above)
Comment (noveogroup-amorgunov) [reply README.md]: Fixed 27d7518165bc794051ef17738f2f31da00e5748d

**Comment by noveogroup-amorgunov** [comment]
@illright thanks for fast review, I fixed all comments

**Review by illright — APPROVED** [review]

**Pull request merged**